lib/katello/permission_creator.rb

```diff
@@ -51,11 +51,16 @@ def activation_keys_permissions
     end
 
     def capsule_content_permissions
+      @plugin.permission :view_smart_proxies,
```

Comment:
Thoughts on using view_capsule_content permission to manage this access instead of the new permission?

Reply:
Let me check with Vlad and if approves then I will move it.

Reply:
Vlad said this:

Reply:
@sjha4 I am ok with whatever you want to roll with based on Vlad's feedback

Reply:
I am still leaning towards the old permission. Let's check with @parthaa ..

Reply:
@sjha4 updated